dev-docs/modules/wurflRtdProvider.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page_v2
+title: WURFL RTD Provider Module
+display_name: WURFL RTD Module
+description: WURFL Real Time Data Module
+page_type: module
+module_type: rtd
+module_code : wurflRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+{: .alert.alert-warning :}
+This module loads a dynamically generated JavaScript from prebid.wurflcloud.com
+
+# WURFL RTD Submodule
+
```

Comment:
please remove the space between the header and the no_toc

Reply:
updated in c23cec5